clang/test/Driver/sycl-offload.cpp

```diff
@@ -0,0 +1,9 @@
+/// Check "-aux-target-cpu" and "target-cpu" are passed when compiling for SYCL offload device and host codes:
+//  RUN:  %clang -### -fsycl -c %s 2>&1 | FileCheck -check-prefix=CHECK-OFFLOAD %s
+//  CHECK-OFFLOAD: "-aux-target-cpu" "x86-64"
+//  CHECK-OFFLOAD: "-target-cpu" "x86-64"
+
+/// Check "-aux-target-cpu" and "-target-cpu" are passed when compiling for SYCL offload device and host codes with enabling macros:
```

Comment:
`enabling macros` has nothing to do with this check.  Probably shouldn't be mentioned.

Reply:
Thanks @mdtoguchi for the feedback. I have updated the patch.